disposable_email_blacklist.conf

```diff
@@ -66,6 +66,7 @@
 20email.it
 20mail.in
 20mail.it
+20mail.eu
```

Comment:
Shouldn't it be before `20mail.in`, alphabetically speaking?

Reply:
indeed it should, will change that